Protacon.RxMq.AzureServiceBus/AzureMqSettingsBase.cs

```diff
@@ -13,5 +13,8 @@ public abstract class AzureMqSettingsBase
         public string AzureResourceGroup { get; set; }
         public string AzureNamespace { get; set; }
         public string AzureSubscriptionId { get; set; }
+        public string AzureRetryMinimumBackoff { get; set; }
+        public string AzureRetryMaximumBackoff { get; set; }
+        public string AzureMaximumRetryCount { get; set; }
```

Comment:
It would be better to use integers here because parsing of configurations should not be the responsibility of `AzureTopicPublisher`. I would also advice updating netcoreapp to newer version so we can utilize newer version of Microsoft.Extensions.Configuration which has configuration `.Bind`

This would also be a good opportunity to fix part of this https://github.com/by-pinja/Protacon.RxMq/issues/4

Reply:
Maybe parse the string in the setter to int value with reasonable default values?

Reply:
Yes, that would be good solution for this which doesn't require upgrading netcoreapp.

Reply:
Ah, actually I missunderstood. AzureMqSettingsBase.cs should not do the parsing. Parsing should be the responsibility of the user of this library.